website/src/components/Messages.tsx

```diff
@@ -7,14 +7,15 @@ import { FlaggableElement } from "./FlaggableElement";
 export interface Message {
   text: string;
   is_assistant: boolean;
+  message_id: string;
 }
 
 export const Messages = ({ messages, post_id }: { messages: Message[]; post_id: string }) => {
   const items = messages.map((messageProps: Message, i: number) => {
     const { text } = messageProps;
-
+    const { message_id } = messageProps;
```

Comment:
could you merge with the line above?

Reply:
sure

Reply:
Did you mean ? 
`const { message_id } = messageProps;
    const { text } = messageProps;
`